server/utils/PluginManager.ts

```diff
@@ -52,83 +53,33 @@ export type Plugin<T extends PluginType> = {
   description?: string;
   /** The plugin content */
   value: PluginValueMap[T];
-  /** An optional priority, will affect order in menus and execution. Lower is earlier. */
-  priority?: number;
-  /** Whether the plugin is enabled (default: true) */
-  enabled?: boolean;
+  /** Priority will affect order in menus and execution. Lower is earlier. */
+  priority: number;
 };
 
 export class PluginManager {
   private static plugins = new Map<PluginType, Plugin<PluginType>[]>();
-
   /**
-   * Register a plugin of a given type.
-   *
-   * @param type The plugin type
-   * @param value The plugin value
-   * @param options Additional options, including whether the plugin is enabled and it's priority.
-   * @returns The PluginManager instance, for chaining.
+   * Add plugins
+   * @param plugins
    */
-  public static register<T extends PluginType>(
-    type: T,
-    value: PluginValueMap[T],
-    options: Omit<Plugin<T>, "value"> = {
-      id: uuid(),
-    }
-  ) {
-    if (!this.plugins.has(type)) {
-      this.plugins.set(type, []);
+  public static add(plugins: Array<Plugin<PluginType>>) {
```

Comment:
Suggest we make this so that it can accept an array or singular `Plugin` as singular is a very common case.